src/Thujohn/Twitter/TwitterServiceProvider.php

```diff
@@ -62,4 +62,4 @@ public function provides()
 		return ['ttwitter'];
```

Comment:
It probably doesn't even matter actually, as this Service Provider isn't `deferred`